### DISCUSSION TITLE
Is it possible to make author image not round and not resize authomatically?

### DISCUSSION THREAD

**Discussion by plovlover** [jpanther/congo#271] [Q&A]
I have an image thats 600x200 px and want to make it as my author image, but by default it transforms into round and resizes automatically? Is there anything I can make to disable it?

**Comment by jpanther** [comment]
The way to do this is by providing a custom homepage layout. You can then simply remove the `rounded-full` class from the image and specify new image dimensions to fit your desired image size.

The [theme docs](https://jpanther.github.io/congo/docs/homepage-layout/#custom-layout) have more details on how you can provide your own layout.
